docs/fragments/pr-sanitizer.adoc

```diff
@@ -2,9 +2,11 @@
 
 This plugin enforces writing effective pull requests that confirm with proper naming conventions and description to clearly convey its purpose and provide better collaboration, traceability and maintainability by giving a clear and concise context to the reviewer of what’s being solved at a high level, taking off an extra step for the reviewer.
 
+If any of the `pr-sanitizer` checks finds out that the pull request doesn't comply with any of the predefined conventions then it marks the plugin's check as **Failure**, or as **success** otherwise.
+
 === Title Verification [[title-verification]]
 
-This ensures that the PR title comforms with the semantic commit message style. Conformance with the semantic commit message style not only makes your changelog and git history clean and easy to navigate but also encourages making atomic pull requests that address only a single concern, better reflect the change within and are more clearly understood.
+This ensures that the PR title conforms with the semantic commit message style. Conformance with the semantic commit message style not only makes your changelog and git history clean and easy to navigate but also encourages making atomic pull requests that address only a single concern, better reflect the change within and are more clearly understood.
```

Comment:
How about a link to that here too? We can make an alias for `https://seesparkbox.com/foundry/semantic_commit_messages` and link whenever we say "semantic commit message".

Reply:
I added the link for every use case when we mention the semantic commit message:

* general description
* failure status details page
* status message comment

but only once for every page - I tried to use it every time where the  "semantic commit message" phrase is used, but it looks a bit weird - referencing the same source several times in the same paragraph.

7ee236a